pkg/apis/batch/validation/validation.go

```diff
@@ -205,7 +205,11 @@ func validateConcurrencyPolicy(concurrencyPolicy *batch.ConcurrencyPolicy, fldPa
 
 func validateScheduleFormat(schedule string, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
-	_, err := cron.Parse(schedule)
+	tmpSchedule := schedule
+	if len(schedule) > 0 && schedule[0] != '@' {
+		tmpSchedule = "0 " + schedule
+	}
+	_, err := cron.Parse(tmpSchedule)
```

Comment:
I bet @pweil- won't believe seeing this, because usually I'm the one complaining about short ifs, he calls me short if police, even :wink: